_data/students.yml

```diff
@@ -165,3 +165,6 @@
   github: Yash-Garg
   image: yashg.png
 
+- name: 三銃士
```

Comment:
Can you make this up in English pls?

Reply:
I though UTF-8 character can be shown on the website?

Reply:
It can, but if you could add it to English it would make it world readable.

Reply:
Finished.

Reply:
Could you merge the commits for me or show me how to merge the commits?